devel/cutter-r2/Portfile

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           qmake5 1.0
+
+github.setup        radareorg cutter 1.10.3 v
+name                cutter-r2
+revision            0
+
+categories          devel
+platforms           darwin
+license             GPL-3
+maintainers         {@i0ntempest me.com:szf1234} openmaintainer
+
+description         Free and Open Source RE Platform powered by radare2
+long_description    Cutter is a free and open-source reverse engineering framework\
+                    powered by radare2. Its goal is making an advanced, customizable\
+                    and FOSS reverse-engineering platform while keeping the user\
+                    experience at mind. Cutter is created by reverse engineers for\
+                    reverse engineers.
+
+homepage            https://cutter.re/
+
+checksums           rmd160  e2c88f2eb32c0ed6cee9e8cd3bc71ddf69363ed3 \
+                    sha256  4a65e5c2a02e0dd7d7359ede2f0ebb5c9d0e3411d6718f6aad57ba9affe53717 \
+                    size    1866181
+
+depends_lib-append  port:radare2
+qt5.depends_component \
+                    qtsvg
+                    
+compiler.cxx_standard 2011
+configure.args-append \
+                    src/Cutter.pro \
+                    INCLUDEPATH+=${prefix}/include/libr
+
+destroot {
+        copy ${worksrcpath}/Cutter.app ${destroot}${applications_dir}
+}
+
+variant python38 description {Enable Python support and bindings using Python 3.8} {
```

Comment:
There's lots of occurrences of `3.8` and `38` inside this variant's code. You could make it simpler to update this to a newer python version later, or to add multiple python variants, by using a variable for that number instead.